tests/benchmark/factors/factors.js

```diff
@@ -3,13 +3,14 @@
 // which makes more faster than other bytecode interpreted
 // VM language listed here.
 
-var start = +new Date();
+var start = process.hrtime()
 
 var N = 50000000;
-var factors = []
-for (var i = 0; i <= N; i++) {
-	if (N % i == 0) factors.push(i);
+var i=0, factors = []
+for (; i <= N; i++) {
+	if (N % i === 0) factors.push(i);
 }
 
-var end = +new Date();
-console.log('elapsed: ' + (end - start)/1000 + 's');
+var end = process.hrtime(end);
```

Comment:
shouldn't this be `var end = process.hrtime(start);` ?

Reply:
Yup, thanks. Didn't run this one